openreview/conference/templates/paperRankingWebfield.js

```diff
@@ -0,0 +1,117 @@
+// ------------------------------------
+// Add Bid Interface
+// ------------------------------------
+
+var CONFERENCE_ID = '';
+var BLIND_INVITATION_ID = CONFERENCE_ID + '/-/Blind_Submission';
+var ANONREVIEWER_WILDCARD = CONFERENCE_ID + '/Paper.*/AnonReviewer.*';
+var PAPER_RANKING_ID = '';
+var GROUP_NAME = '';
+
+var INSTRUCTIONS = '<p class="dark">Please rank your assigned papers selecting a number in each dropdown below where value 1 is the best.</p>\
+<br>'
+
+var getNumberfromGroup = function(groupId, name) {
+  var tokens = groupId.split('/');
+  var paper = _.find(tokens, function(token) {
+      return _.startsWith(token, name);
+  });
+
+  if (paper) {
+    return parseInt(paper.replace(name, ''), 10);
+  } else {
+    return null;
+  }
+};
+
+var getPaperNumbersfromGroups = function(groups) {
+  return _.filter(_.map(groups, function(group) {
+    return getNumberfromGroup(group.id, 'Paper');
+  }), _.isInteger);
+};
+
+// Main is the entry point to the webfield code and runs everything
+function main() {
+  Webfield.ui.setup('#invitation-container', CONFERENCE_ID);  // required
+
+  Webfield.ui.header(GROUP_NAME + ' Paper Ranking Console', INSTRUCTIONS);
+
+  Webfield.ui.spinner('#notes', { inline: true });
+
+  load().then(renderContent);
+}
+
+
+// Perform all the required API calls
+function load() {
+  var notesP = Webfield.getAll('/groups', {member: user.id, regex: ANONREVIEWER_WILDCARD})
+  .then(function(groups) {
+    return getPaperNumbersfromGroups(groups);
+  })
+  .then(function(noteNumbers) {
+    var noteNumbersStr = noteNumbers.join(',');
+
+    return Webfield.get('/notes', { invitation: BLIND_INVITATION_ID, number: noteNumbersStr, details: 'tags' })
+      .then(function(result) {
+        return result.notes;
+      });
+  });
+
+  var tagInvitationsP = Webfield.getAll('/invitations', {id: PAPER_RANKING_ID, type: 'tags'}).then(function(invitations) {
+    return invitations.filter(function(invitation) {
+      return invitation.invitees.length;
+    });
+  });
+
+  return $.when(notesP, tagInvitationsP);
+}
+
+
+// Display the bid interface populated with loaded data
+function renderContent(notes, tagInvitations) {
+
+  var paperDisplayOptions = {
+    pdfLink: true,
+    replyCount: true,
+    showContents: true,
+    showTags: true,
+    tagInvitations: tagInvitations
+  };
+
+  var sections = [
+    {
+      heading: 'Assigned Papers',
+      id: 'allPapers',
+      content: null
+    }
+  ];
+  sections[0].active = true;
+
+  $('#notes .tabs-container').remove();
+
+  Webfield.ui.tabPanel(sections, {
+    container: '#notes',
+    hidden: true
+  });
+
+  // Render the contents of the All Papers tab
+  Webfield.ui.submissionList(notes, {
```

Comment:
@zbialecki Does submissionList has an option to pass the 'referrer' value? I forgot to add that.

Reply:
No, submissionList currently doesn't support that, but I can add it in my other assignments list page PR.

Reply:
It's added here: https://github.com/openreview/openreview/pull/1919
You can add the referrer by including it in the `displayOptions` like, 
```
Webfield.ui.submissionList(notes, {
  ...
  displayOptions: { referrer: 'referrer string here' }
}
```